sollumz_preferences.py

```diff
@@ -547,7 +560,7 @@ def _update_bpy_struct_from_dict(struct: bpy_struct, values: dict, eval_strings:
         if value is None:
             continue
 
-        if eval_strings and isinstance(value, str):
+        if eval_strings and isinstance(value, str) and key != "default_game":
```

Comment:
This check is because of the parsing error when storing strings in the preferences .ini? It was recently fixed on upstream: https://github.com/Sollumz/Sollumz/blob/fbb8248d0970129500cbd2d15aabb137699a8284/sollumz_preferences.py#L576